contracts/ClaimsManager.sol

```diff
@@ -11,13 +11,9 @@ contract ClaimsManager is
     IClaimsManager
 {
     struct Claim {
-        bytes32 policyHash;
         ClaimStatus status;
-        address claimant;
-        address beneficiary;
         uint32 updateTime;
-        uint256 amountInUsd;
-        string evidence;
+        bytes27 hash_;
```

Comment:
I would probably use a more descriptive name for this var. Something like claimDetailsHash

Reply:
This field should go away with https://github.com/api3dao/claims-manager/pull/50#issuecomment-1225911495 anyway